src/components/header/__test__/header.test.js

```diff
@@ -0,0 +1,32 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { HashRouter } from 'react-router-dom';
+import { toMatchDiffSnapshot } from 'snapshot-diff';
+import Header from '../Header';
+import '@testing-library/jest-dom';
+
+expect.extend({ toMatchDiffSnapshot });
+
+test('@Header Rocket Navigation', () => {
+  const { asFragment } = render(<HashRouter><Header /></HashRouter>);
+  fireEvent.click(screen.getByText(/Rocket/));
+  const firstRender = asFragment();
+  fireEvent.click(screen.getByText(/Rocket/));
+
+  expect(firstRender).toMatchDiffSnapshot(asFragment());
+});
```

Comment:
- [ ] Good job writing all the tests, however, some tests failed so I attached the image of the failing tests below. Kindly fix the tests to ensure all the tests passed.
<img width="402" alt="chrome_ENadtse8ED" src="https://user-images.githubusercontent.com/81468675/168059961-86168eb4-2a27-4193-bf94-383597f9bb7d.png">
<img width="518" alt="chrome_CuP0at94Rf" src="https://user-images.githubusercontent.com/81468675/168060000-baab5723-441b-4a1a-86a1-dfff3028d8b0.png">